src/pages/NotFound/NotFound.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import NotFoundPage from "./NotFound.page";
+
+describe("<NotFoundPage>", () => {
+  test("should create the NotFound Page correctly", () => {
+    const { getByTitle } = render(<NotFoundPage />);
+    const text = getByTitle("title");
+    expect(text).toBeInTheDocument();
+  });
+});
```

Comment:
This test technically fulfills the requirements, yet this test could be improve by testing that when we put a not valid route, the browser will render this page.